config/routes/spree.rb

```diff
@@ -1,3 +1,13 @@
+Openfoodnetwork::Application.routes.draw do
+  scope module: 'spree' do
+    resources :orders do
+      get :clear, :on => :collection
+      get :order_cycle_expired, :on => :collection
+      put :cancel, on: :member
+    end
```

Comment:
while we exercise `put :cancel` in https://github.com/openfoodfoundation/openfoodnetwork/blob/9fe2cb1a3042d63ea44e73710341e5d0511cf812/spec/controllers/spree/orders_controller_spec.rb#L409-L448. `git grep "spree_get :order_cycle_expired"` and `git grep  "spree_get :clear"` return no results. I think it's worth adding the simplest test we can think of for each to ensure the routes are hit. That would give me the confidence to merge this upon a green build. Thoughts?

Reply:
well, looks like dead code to me... I deleted both and the build is green.
I cant tell where this could be used :man_shrugging:

Reply:
Lol, that's right :see_no_evil: It seems that we stopped using `order_cycle_expired` ages ago (https://github.com/openfoodfoundation/openfoodnetwork/commit/5f712a898, https://github.com/openfoodfoundation/openfoodnetwork/commit/151702d68) and no trace of any `order/:id/cancel`.